AraCorrelator/RayTraceCorrelator.cxx

```diff
@@ -402,81 +482,57 @@ TH2D RayTraceCorrelator::GetInterferometricMap(
         throw std::invalid_argument(errorMessage);
     }
 
-    TH2D histMap("", "", this->numPhiBins_, -180, 180, this->numThetaBins_, -90, 90);
+    const int numGlobalBins = int((this->numThetaBins_+2)*(this->numPhiBins_+2));
+
+    // a variable to store the summed correlation value
+    // this needs to have the same size as the eventual TH2D
+    std::vector<double> summedCorr(numGlobalBins, 0);
+
+    // and a variable to control the loop over bins we actually populated
+    // this is NOT the same thing as the number of bins in the TH2D
+    // since the number of bins in the 2D hist is different than the number of bins we have cached
+    // because of overflow bins
```

Comment:
and underflow bins. Might as well mention haha.

Reply:
Roger.